翻译曲线编辑器及自定义构建模板 (#1926)
cocos-creator/creator-docs: en/animation/curve-editor.md

- `Right`: right tangent is in in weighting mode, left tangent is in no-weighting mode.
- `Both`: both left and right tangents are in weighting mode.

#### Broken

By default, the left and right tangent slopes of the keyframe are kept the same. Adjusting the tangent lever on either side will follow the adjustment on the other side, which makes it easier for us to adjust the transition more naturally. In some cases, the curve does not always transition smoothly. At this time, right-click the keyframe and click the Broken option in the right-click menu to disconnect the key left and right joysticks. After disconnecting, the keyframes can be adjusted separately using the left and right joysticks.

> 

Reviewer: ```suggestion
```

> By default, the left and right tangent slopes of the keyframe are kept the same. Adjusting the tangent lever on either side will follow the adjustment on the other side, which makes it easier for us to adjust the transition more natural curve. But in some cases, we hope that the curve does not always transition smoothly. At this time, we can right-click the keyframe and click the Broken option in the right-click menu to disconnect the key left and right joysticks. After disconnecting, the keyframes can be adjusted separately The left and right joysticks.

Reviewer 2: ```suggestion
By default, the left and right tangent slopes of the keyframe are kept the same. Adjusting the tangent lever on either side will follow the adjustment on the other side, which makes it easier for us to adjust the transition more naturally. In some cases, the curve does not always transition smoothly. At this time, right-click the keyframe and click the Broken option in the right-click menu to disconnect the key left and right joysticks. After disconnecting, the keyframes can be adjusted separately using the left and right joysticks.
```
